core/blockchain.go

```diff
@@ -162,6 +163,10 @@ func NewBlockChain(chainDb ethdb.Database, pow pow.PoW, mux *event.TypeMux) (*Bl
 	return bc, nil
 }
 
+func (self *BlockChain) SetConfig(vmConfig *vm.Config) {
```

Comment:
Don't worry about this bit. It's removed in https://github.com/ethereum/go-ethereum/pull/2281 I don't expect the two to take very long to get merged after eachother. It's just a quick workaround to enable jit. We could drop this commit all together if that makes more sense.